### DISCUSSION TITLE
Having trouble downloading MagickWand

### DISCUSSION THREAD

**Discussion by khuangpu** [jsvine/pdfplumber#1107] [Q&A]
Trying to go through the pdfplumber tutorial notebook and I keep on getting stuck when trying to download MagickWand when I get to the part where I try to look at the pdf image.

The specific error is: 
ImportError: MagickWand shared library not found.
You probably had not installed ImageMagick library.
Try to install:
  brew install freetype imagemagick

**Comment by jsvine** [comment]
Hi, and thanks for your interest! MagicWand is no longer part of `pdfplumber`'s requirements. Is it possible that you have an older version of `pdfplumber` installed? One way to fix/check this is by running `pip install -U pdfplumber`. Then, when you run `python -c 'import pdfplumber; print(pdfplumber.__version__)'`, what do you see?

**Comment by khuangpu** [comment]
That was the issue! I ran that first line of code in the terminal and then
went back to my Jupyter Notebook and the tutorial ran through without any
errors.  Thank you so much!
